BCStudentSoftwareDevTeam/celts
# Admin Management Page

Issue: There needs to be an Admin Management page that allows the admin to add users as Celts Admin or Celts Student Staff. Admin should be able to remove a user from being a Celts Admin or Celts Student Staff. Only Admin has access to this page.
![Admin Management Page](https://user-images.githubusercontent.com/59736834/136981046-b73af7c8-36a8-4451-9823-99bc505447da.jpg)
Test: 
1. Add a valid user who's not a Celts Admin as a Celts Admin. After clicking **Add**, a success flash message should pop up saying they are a Celts Admin now. In the database, user.isCeltsAdmin = 1. Then, remove admin privilege from them; there should also be a green flash message and user.isCeltsAdmin = 0.
2.  Add a valid user who's not a Celts Student Staff as a Celts Student Staff. After clicking **Add**, a success flash message should pop up saying they are a Celts Student Satff now. n the database, user.isCeltsStudentStaff = 1. Then, remove Celts Student Satff privilege from that same user. A green flash message should appear and user.isCeltsStudentStaff = 0. 
3. reset database and run tests; all the tests should pass.
